src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts.
+category: Assistive technology
+author: Hamsa Harcourt
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+> Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence
+
+Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.
+
+
+## Case study
+
+When designing fonts, it's common practice to flip and rotate a character to form other characters. Letters b, p, d and q are usually created using this method. However, this causes problems for people with dyslexia. This is because it makes the letters easy to swap. As a result, people with dyslexia prefer fonts where all the characters are unique. 
+
+
+## Introducing dyslexia fonts
+
+Dyslexia fonts are typefaces designed specially for people with dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts make each character unique by using thicker lines in parts of the letters. Also, the letters are slanted a bit.  Some dyslexia fonts include: Dyslexie, Open dyslexic, Gill dyslexic, Read Regular etc.
+
+![an example of dyslexia fonts](/img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)
+ 
+However, according to recent studies, it has been proven that dyslexia fonts do **not** help dyslexics **read faster and make fewer mistakes**. In fact, most dyslexics prefer Verdana and Helvetica to  other fonts , including dyslexia fonts.
+
+
+## General font tips
+
+The following are general font tips that improves the overall quality of a website:
+
+- Font size should be at least 12-14pt or its equivalent.
+- Larger line spacing improves readability.
+- Use a small number of fonts, ideally 1 or 2.
+- Use **bold** to add emphasis rather than *italics* or UPPERCASE.
+- Animated texts must be used sparingly.
+
+
+## Summary
+
+Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for dyslexia. This is because dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.
```

Comment:
```suggestion
Dyslexia is characterized as a condition where people transpose characters. Using a specific font is not a treatment for Dyslexia. This is because Dyslexia is an issue with language, not vision. However, sans serif fonts have proven easiest to read on screen.
```